interface/themes/style.php

```diff
@@ -0,0 +1,68 @@
+<?php 
+/** 
+ * function to apply selected color in the color picker
+ * 
+ * Copyright (C) 2017 Art Eaton
```

Comment:
Change this to 2018 and Your name

Reply:
How the heck did my name get in there???